src/main/java/com/google/swarm/tokenization/common/CSVFileReaderSplitDoFn.java

```diff
@@ -52,7 +52,11 @@ public void processElement(ProcessContext c, RestrictionTracker<OffsetRange, Lon
           new FileReader(
               channel, tracker.currentRestriction().getFrom(), recordDelimiter.getBytes());
       while (tracker.tryClaim(reader.getStartOfNextRecord())) {
+        Long readerPosition = reader.getStartOfNextRecord();
```

Comment:
```suggestion
        long readerPosition = reader.getStartOfNextRecord();
```
Please use primitive type instead of boxed type.